.build/iconfont.scss

```diff
@@ -40,9 +40,12 @@ $ti-prefix: 'ti' !default;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+@function unicode($str) {
+  @return unquote("\"")+unquote(str-insert($str, "\\", 1))+unquote("\"")
+}
 
 <% glyphs.forEach(function(glyph) { %>
-$ti-icon-<%= glyph.name %>: '\<%= glyph.unicode[0].codePointAt(0).toString(16) %>';<% }); %>
+$ti-icon-<%= glyph.name %>: unicode('<%= glyph.unicode[0].codePointAt(0).toString(16) %>');<% }); %>
```

Comment:
why does it have to be unicode encoded? It works fine without it ;)

Reply:
Because you can not see to which character it actually maps, you just see a special character which is for every icon the same.

Reply:
I get it. Let me test if it works as before over the weekend and get back to you here :)